escapeRoom/build.gradle

```diff
@@ -5,10 +5,16 @@ plugins {
 
 dependencies {
     api project(':dungeon')
+
+    // JUnit and Mockito for testing
+    testImplementation supportDependencies.junit
+    testRuntimeOnly supportDependencies.junitLauncher
+    testImplementation supportDependencies.mockito_core
```

Comment:
ich versteh übrigens nicht, warum ich das aus dem Dungeon gradle kopieren musste. 
Das soltle doch mit der dependencie zum Dungeon dabei sein

Reply:
Also Gradle Fragen sollte am Besten @cagix beantworten 😄

Reply:
@AMatutat ist im dungeon nicht als "api" markiert und wird also nur im dungeon genutzt, aber nicht vom dungeon-projekt hochgereicht. 

schaust du in die doku: https://docs.gradle.org/current/userguide/dependency_configurations.html#two-dependency-configurations

es sieht aber flüchtig betrachtet danach aus, als ob die test-dependencies immer nur für das projekt sein können?

Reply:
> Also Gradle Fragen sollte am Besten @cagix beantworten 😄 

@Flamtky bin ich ne verdammte wandelnde doku? 🫣🤣 ... die anlaufstelle sollte eigentlich immer die doku sein

Reply:
Vergesst `man` ic nutz `cagix`